app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -33,20 +26,14 @@ def initialize(solr_document, current_ability, request = nil)
       @request = request
     end
 
+    # We cannot rely on the method missing to catch this delegation.  Because
+    # most all objects implicitly implicitly implement #to_s
```

Comment:
So nice you can say it twice (I'm guessing this was accidental) 

```suggestion
    # most all objects implicitly implement #to_s
```